### PR TITLE
Update dependencies, licenses and maintainers

### PR DESCRIPTION
**Pull request by t20100**
- Adding optional dependencies
- Updating license
- Adding maintainers

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by t20100** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,7 +38,7 @@ test:
 
 about:
     home: https://github.com/silx-kit/fabio
-    license: GPL v3.0, GPL v2.0, MIT, GPL, LGPL v3.0, BSD 3-Clause, Apache 2.0
+    license: MIT, LGPL v3.0
```
Comment (CJ-Wright) [inline recipe/meta.yaml:42]: I think this can actually be provided as a yaml list

Comment (t20100) [reply recipe/meta.yaml:42]: Linting doesn't like it...

Comment (CJ-Wright) [reply recipe/meta.yaml:42]: hmm, ok I guess we need to fix the linting at some point

**Comment by t20100** [comment]
@conda-forge-admin, please rerender

**Comment by t20100** [comment]
Thanks for the fix, I was wondering what was going wrong.

**Comment by tschoonj** [comment]
Looks good now I think... can I merge it in or do you need to make any other changes?

**Comment by t20100** [comment]
You can merge, it is fine for me.
Thanks!

**Pull request merged**